guide/14-deep-learning/How CycleGAN Works.ipynb

```diff
@@ -0,0 +1,198 @@
+{
```

Comment:
<p>"There are some challenges in Pix2Pix is that the images should be paired which are randomly organized or unpaired and we want to convert images from one domain to another just like " -&gt; <strong>this sentence seems incomplete. Further, can you elaborate or reword the limitations of Pix2Pix model?</strong></p><p> </p><p><strong>I think we also need to elaborate what you mean by an unpaired dataset</strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/845/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Yes, the sentence is incomplete. Thanks for pointing it out .